financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/FinancialConnectionsDataExampleActivityJava.java

```diff
@@ -0,0 +1,77 @@
+package com.stripe.android.financialconnections.example;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.lifecycle.ViewModelProvider;
+
+import com.stripe.android.financialconnections.FinancialConnections;
+import com.stripe.android.financialconnections.FinancialConnectionsSheet;
+import com.stripe.android.financialconnections.example.FinancialConnectionsExampleViewEffect.OpenFinancialConnectionsSheetExample;
+
+import kotlin.Unit;
+
+public class FinancialConnectionsDataExampleActivityJava extends AppCompatActivity {
+
+    private FinancialConnectionsExampleViewModel viewModel;
+    private FinancialConnectionsSheet financialConnectionsSheet;
+
+    private TextView status;
+    private Button launchConnectionsSheet;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_financialconnections_example);
+        viewModel = new ViewModelProvider(this).
+                get(FinancialConnectionsExampleViewModel.class);
+        setupViews();
+        observeViews();
+        observeState();
+        FinancialConnections.setEventListener(
+                event -> {
+                    Log.d("FinancialConnections", "Event: " + event.getName());
+                    return Unit.INSTANCE;
```

Comment:
This kinda sucks :/

Reply:
Yeah this is unexpected. Is there no workaround to avoid needing to add that?

Reply:
Nice!